notification2-examples/java-examples/src/main/java/c8y/example/notification/samples/Example1.java

```diff
@@ -0,0 +1,65 @@
+package c8y.example.notification.samples;
+
+import c8y.example.notification.common.websocket.WebSocketClient;
+import com.cumulocity.model.idtype.GId;
+import com.cumulocity.rest.representation.inventory.ManagedObjectRepresentation;
+import com.cumulocity.rest.representation.reliable.notification.NotificationSubscriptionFilterRepresentation;
+import com.cumulocity.rest.representation.reliable.notification.NotificationSubscriptionRepresentation;
+
+import java.util.List;
+
+/**
+ * Multiple “subscriptions” using the same topic (i.e., same subscription name)
+ */
+public class Example1 extends Notification2Example {
```

Comment:
favour composition over inheritance

Reply:
Agree with the above. The current base class can be a separate 'Util' class and it looks like it's methods can be static (so we don't even need an instance of it).

Reply:
👍

Reply:
Done.